apps/music/index.html

```diff
@@ -97,16 +97,16 @@ <h1 id="title-text" data-l10n-id="music">Music</h1>
           <div id="player-cover-caption">
             <div id="player-cover-artist" data-l10n-id="songArtist">Song Artist</div>
             <div id="player-cover-album" data-l10n-id="songAlbum">Song Album</div>
-            <button id="player-cover-share" class="player-cover-button" data-icon="share"></button>
+            <button id="player-cover-share" class="player-cover-button" data-icon="share" data-l10n-id="share-song"></button>
           </div>
           <div id="player-cover-buttons">
             <button id="player-album-repeat" class="player-cover-button" data-icon="repeat"></button>
-            <div id="player-album-rating">
-              <button class="rating-star" data-rating="1"></button>
-              <button class="rating-star" data-rating="2"></button>
-              <button class="rating-star" data-rating="3"></button>
-              <button class="rating-star" data-rating="4"></button>
-              <button class="rating-star" data-rating="5"></button>
+            <div id="player-album-rating" data-l10n-id="player-album-rating" role="region" aria-live="polite">
+              <button class="rating-star" data-rating="1" role="checkbox" data-l10n-id="rating-star-1"></button>
```

Comment:
We can be a little more fancy here, we can use a single data-l10n-id="rating-start" with another attribute that's different for each radiobutton - data-l10n-args='{"n": "1"}' for one start, data-l10n-args='{"n": "2"}' for two, etc.